screens/EditBaniOrder.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable max-classes-per-file */
```

Comment:
Veerji, by any chance can we remove these disabled rules lines?

You can take help from the below links
- https://eslint.org/docs/latest/rules/no-underscore-dangle
- https://eslint.org/docs/latest/rules/max-classes-per-file